dev-python/pyroute2/pyroute2-0.5.19.ebuild

```
@@ -0,0 +1,31 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{7,8,9,10} pypy3 )
+PYTHON_REQ_USE="sqlite"
+DISTUTILS_USE_SETUPTOOLS=bdepend
+
+inherit distutils-r1
+
+DESCRIPTION="A pure Python netlink and Linux network configuration library"
+HOMEPAGE="https://github.com/svinota/pyroute2"
+SRC_URI="https://github.com/svinota/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
```

```suggestion
SRC_URI="
	https://github.com/svinota/pyroute2/archive/${PV}.tar.gz
		-> ${P}.gh.tar.gz
"
```

pkgcheck complains about bad filename these days.